onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -5907,19 +5907,137 @@ def aten_std_mean(self: TensorType, unbiased: bool = True) -> tuple[TensorType,
     raise NotImplementedError()
 
 
+@torch_op("aten::stft", private=True)
+def _add_batch_dimension(self: TFloatOrBFloat16) -> tuple[TFloatOrBFloat16, INT64]:
```

Comment:
```suggestion
def _add_batch_dimension(self: TFloatOrBFloat16) -> Tuple[TFloatOrBFloat16, INT64]:
```

this will fix the python38 error

Reply:
THanks